From: =?utf-8?q?Barnab=C3=A1s_P=C5=91cze?= <pobrn@protonmail.com>
Date: Wed, 22 Nov 2023 12:05:47 +0000
Subject: Apply 1 suggestion(s) to 1 file(s)

(cherry picked from commit abc4bd111be6273298aaeb290d8fb613ac7316d9)
Origin: upstream, after 1.0.1
---
 src/modules/module-protocol-pulse/snap-policy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/module-protocol-pulse/snap-policy.c b/src/modules/module-protocol-pulse/snap-policy.c
index d838d33..a6f2e24 100644
--- a/src/modules/module-protocol-pulse/snap-policy.c
+++ b/src/modules/module-protocol-pulse/snap-policy.c
@@ -58,7 +58,7 @@ pw_sandbox_access_t pw_snap_get_audio_permissions(struct client *client, int fd,
     SnapdPlug **plug = NULL;
     GPtrArray *slots = NULL;
     SnapdSlotRef **slot = NULL;
-    GError *error = NULL;
+    g_autoptr(GError) error = NULL;
     int exit_code;
 
     *app_id = g_strdup("unknown");
